examples/storage/s3/incremental_learning/README.md

```diff
@@ -1,10 +1,15 @@
 # Using Incremental Learning Job in Helmet Detection Scenario on S3
 
-This example based on the example: [Using Incremental Learning Job in Helmet Detection Scenario](/examples/incremental_learning/helmet_detection/README.md)
+This example based on the example: [Using Incremental Learning Job in Helmet Detection Scenario](/examples/incremental_learning/helmet_detection/README.md).    
 
-### Create a secret with your S3 user credential.
+### Prepare Nodes
+Assume you have created a [KubeEdge](https://github.com/kubeedge/kubeedge) cluster that have two cloud nodes(e.g., `Cloud_Node1`, `Cloud_Node2`)

```

Comment:
The name of a Node object must be a valid DNS subdomain name https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names

 This means the name must:
- contain no more than 253 characters
- contain only lowercase alphanumeric characters, '-' or '.'
- start with an alphanumeric character
- end with an alphanumeric character

Reply:
fix